www/site/content/docs/chef-workstation/chef-run-users-guide.md

```diff
@@ -78,15 +72,16 @@ the resource type in the second place, and the resource name in the third
 place. For example:
 
 ```bash
-chef-run host1 group the_avengers
+chef-run host1 group awesome_group
 ```
 
-The command above specifies the `group` resource with a name of `the_avengers`.
+The command above specifies the `group` resource with a name of `awesome_group`.
```

Comment:
directional words like above, below, left, and right create poor a11y experiences since this is in a section I think you can just drop the _above_, so "This command specifies..."